OracleWebCenterSites/dockerfiles/12.2.1.4/sites-container-scripts/startAdminServer.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Description: This script is used to start a WebLogic Admin server. And tail the logs.
+#
+
+export DOMAIN_NAME=$DOMAIN_NAME
+export DOMAIN_ROOT="/u01/oracle/user_projects/domains"
+export DOMAIN_HOME="${DOMAIN_ROOT}/${DOMAIN_NAME}"
+
+# Start Admin server
+cd $DOMAIN_HOME/bin
+
+echo ""
+echo "tail $DOMAIN_HOME/bin/weblogic.out in a new window"
```

Comment:
This won't work. You can't start a new window to tail `weblogic.out` because it's inside a container. You need to this to be output to `stdout` so that it's visible via `docker logs`